### ISSUE TITLE
What't the Training loss

### ISSUE THREAD

**Issue by Awenjie10** [SeungyounShin/CAGFace#4]
x2 and x4 should be double loss functions in Paper, I just found one. Can you show the place? Thank you
